safe/gui/widgets/profile_widget.py

```diff
@@ -0,0 +1,137 @@
+# coding=utf-8
+"""InaSAFE Profile Widget."""
+
+from PyQt4.QtGui import (
+    QTreeWidget, QTreeWidgetItem, QCheckBox, QDoubleSpinBox, QFont)
+from PyQt4.QtCore import Qt
+from safe.definitions.utilities import get_name, get_class_name
+from safe.utilities.i18n import tr
+from collections import OrderedDict
+
+__copyright__ = "Copyright 2017, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+
+class ProfileWidget(QTreeWidget, object):
+
+    """Profile Widget"""
```

Comment:
My regex will fix that, I don't care now ;-)

Reply:
Yeay.